bin/controller.keys.js

```diff
@@ -28,6 +28,8 @@ var _ = require('lodash');
 module.exports.updateKeys = function updateKeys(options, taskCallback) {
 
     let roles = process.env.ALLOW_SSH_ACCESS_ROLES || "admin,maintain,write";
+    let productionBranch = process.env.PRODUCTION_BRANCH || "production";
+    let adminRole = process.env.ADMIN_ROLE || "admin";
```

Comment:
@kavaribes 
- `ALLOW_PRODUCTION_SSH_ACCES_ROLES`?
- and change it from string to array
what do you think ?

Reply:
```
add ability connect to production containers for Administrators
```
It's from task.
@planv

Reply:
Yes, it is needed in our case.
But we also have to provide ability to expand this value.